examples/catalog/bin/sample_page.dart

```diff
@@ -87,9 +95,11 @@ class SampleInfo {
   // The value of the 'Classes:' comment as a list of class names.
   Iterable<String> get highlightedClasses {
     final String classNames = commentValues['classes'];
-    if (classNames == null)
-      return const <String>[];
-    return classNames.split(',').map<String>((String s) => s.trim()).where((String s) => s.isNotEmpty);
+    if (classNames == null) return const <String>[];
```

Comment:
please put return statements at the start of lines, so that the code flow is more obvious to someone scanning the code (otherwise it looks like this is straight-line code).